include/gdsb/graph_input.h

```diff
@@ -22,6 +22,15 @@ namespace gdsb
 //! to read next from for subsequent reads of integers within one string (stream).
 unsigned long read_ulong(char const* source, char** end = nullptr);
 
+//! Reads in space separated floating point values. Returns infinity if value at
+//! position source can not be interpreted as a number.
+//! @param source   Points to the beginning of the character (sub string) to be
+//! interpreted.
+//! @param end      Points to nullptr on first call, will (point) out position
```

Comment:
this nullptr description is slightly different to the documentation of `std::strtof` - there i says that the end pointer will be ignored if it is a nullptr.

Reply:
I think this is actually in line with what my intention is. If one calls this function for the first time one simply uses `std::nullptr` as `end` which then will be ignored. But it should still return the out position to provide as the next value for `source` if `read_float()` is called consecutively to read multiple floats within one string.

Reply:
Revised the documentation.